### PR TITLE
Problem with fits files using TDIMn = '(1,1)    '

### PR DESCRIPTION
**Pull request by jwoillez**
Starting with a (hopefully) failing test. Fix on its way...

### CODE REVIEW

**Comment by embray** [comment]
You can run the test yourself by running `./setup.py test --package=io.fits` ?  Probably better to have the fix too before pushing the PR since otherwise it's just running a build that is sure to fail :)

**Comment by embray** [comment]
(That aside, thanks for the (upcoming) fix--this definitely looks like a bug!)

**Comment by astrofrog** [comment]
In a way, maybe it's good to push the regression test first to make sure it would fail on Travis and isn't platform-dependent :)

**Comment by jwoillez** [comment]
@embray Thanks for the local test pointer. Apparently, adding `tdim.fits` is sufficient to trigger the issue in `test_hdulist.py`. Will wait for Travis/AppVeyor to fail before pushing a fix.

**Comment by embray** [comment]
I coulda told you this wasn't a platform-dependent issue, but I guess being able to say that with a degree of certainty requires a certain level of experience with the code :p

**Comment by jwoillez** [comment]
Actually, it's the first time that I wish for a little red cross. I just want to see it... :)

**Comment by jwoillez** [comment]
Passes, hence ready for review.

File: astropy/io/fits/tests/test_table.py [hunk 1]
```diff
@@ -1758,6 +1758,10 @@ def test_mismatched_tform_and_tdim(self):
         pytest.raises(VerifyError, fits.Column, name='a', format='2I',
                       dim='(2,2)', array=arra)
 
+    def test_tdim_of_size_one(self):
```
Comment (embray) [inline astropy/io/fits/tests/test_table.py]: If you don't mind, please add a docstring to this test like the one below it linking back to this issue.

**Comment by embray** [comment]
Looks good to me--if you'll just add a changelog entry under the v1.0.2 section (which apparently I need to create, so you may have to rebase your branch first), and this will be good to go.

**Comment by jwoillez** [comment]
Here it is.

**Comment by astrofrog** [comment]
@jwoillez - could you rebase? I think there might be a small conflict in the changelog?

**Comment by embray** [comment]
Huh, sorry, I figured I already merged this one.

**Comment by embray** [comment]
Weird.  For some reason GitHUb didn't create a merge commit for this, and just did a FF.  I hope that's not something they changed on purpose...

**Comment by embray** [comment]
I wonder if I manually merged this, but then later GH added a merge commit for it, except the merge commit is not actually merge commit.  Though there is a merge commit for this at e7c86f9de892c2f27a457162dd050960095ec78b

**Pull request merged**